docs/Inputs.md

```diff
@@ -266,17 +266,17 @@ For instance, let's write a component to edit the latitude and longitude of the
 import { useController } from 'react-hook-form';
 
 const LatLngInput = () => {
-    const input1 = useController({ name: 'lat' });
```

Comment:
I really preferred the previous version

Reply:
For the `LatLngInput ` component, I change a syntax error, what do you mean by the previous version?

Reply:
these 2 lines, you've just removed blank spaces and that makes the code harder to read

Reply:
Sorry you're right